bin/ares-config.js

```diff
@@ -138,25 +139,11 @@ function config() {
 function curConfig(next) {
     const curConfigData = appdata.getConfig(true);
     if (typeof curConfigData.profile === 'undefined') {
-        return finish(errHndl.changeErrMsg("INVALID_VALUE", "profile details"));
+        return finish(errHndl.getErrMsg("INVALID_VALUE", "profile details"));
     } else if (curConfigData.profile.trim() === "") {
-        return finish(errHndl.changeErrMsg("EMPTY_PROFILE"));
+        return finish(errHndl.getErrMsg("EMPTY_PROFILE"));
     } else {
         console.log("Current profile set to " + curConfigData.profile);
         next();
     }
 }
-
-function finish(err, value) {
```

Comment:
Please consider leaving the finish in the bin as it is

Reply:
I moved finish() of errHandle to each bin level command.js file.